### PR TITLE
consul: 1.10.4, 1.9.11 and 1.8.17

### PR DESCRIPTION
**Pull request by eculver**
Latest releases.

### CODE REVIEW

**Comment by tianon** [comment]
```console
+ wget https://releases.hashicorp.com/consul/1.11.0-beta3/consul_1.11.0-beta3_linux_amd64.zip
Connecting to releases.hashicorp.com (151.101.1.183:443)
wget: server returned error: HTTP/1.1 403 Forbidden
```

:eyes:

**Comment by eculver** [comment]
I'm looking into why the artifact for 1.11.0-beta3 has not been published yet.

**Comment by eculver** [comment]
The beta is being excluded from this release.

File: library/consul [hunk 1]
```diff
@@ -6,19 +6,19 @@ GitRepo: https://github.com/hashicorp/docker-consul.git
 GitCommit: 74be3d0c6ce2e12805e0c9be4cf02a44e6f3cec1
 Directory: 0.X
 
-Tags: 1.10.3, 1.10, latest
+Tags: 1.10.4, 1.10, latest
 Architectures: amd64, arm32v6, arm64v8, i386
 GitRepo: https://github.com/hashicorp/docker-consul.git
 GitCommit: e677d8b7151a4e3a994741752b39ae42604cf97d
```
Comment (yosifkit) [inline library/consul:12]: None of these include any Dockerfile changes. I think the `GitCommit's need to be updated.

Comment (eculver) [reply library/consul]: whoops. derp. I was trying to reset around some other rogue diff and forgot to reapply. Thanks. Should be fixed now.

**Pull request merged**